### PR TITLE
C4-38 Test

### PR DESCRIPTION
**Pull request by willronchetti**
- Adds a test that triggers the bug seen in C4-38
- DO NOT MERGE until poetry is merged into FF and Snovault release is made

### CODE REVIEW

File: src/encoded/tests/test_indexing.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
 from ..utils import delay_rerun
 from ..verifier import verify_item
 from .workbook_fixtures import app_settings
+from .test_permissions import wrangler, wrangler_testapp
```
Comment (j1z0) [inline src/encoded/tests/test_indexing.py]: Codacy Issue found: [Unused wrangler_testapp imported from test_permissions](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5008844)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5008844&bid=16522378)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5008844&bid=16522378)

File: src/encoded/tests/test_indexing.py (hunk 1 above)
Comment (j1z0) [inline src/encoded/tests/test_indexing.py]: Codacy Issue found: [Unused wrangler_testapp imported from test_permissions](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5008844)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5008844&bid=16522378)

File: src/encoded/tests/test_indexing.py (hunk 1 above)
Comment (j1z0) [inline src/encoded/tests/test_indexing.py]: Codacy Issue found: [Unused wrangler_testapp imported from test_permissions](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5008844)

File: src/encoded/tests/test_indexing.py [hunk 2]
```diff
@@ -325,3 +326,23 @@ def test_load_and_index_perf_data(testapp, indexer_testapp):
...
+        'paired_end': '1',
+        'status': 'released'
+    }
+    res = wrangler_testapp.post_json('/file_fastq', file_item_body, status=201).json
+    item_id = res['@graph'][0]['@id']
+    indexer_testapp.post_json('/index', {'record': True})
+    sleep(1)  # let es catch up
+    res = anontestapp.get('/' + item_id).json
+    assert res['file_format'] == {'error': 'no view permissions'}
+    res = anontestapp.get('/' + item_id + '?datastore=database').json
```
Comment (netsettler) [inline src/encoded/tests/test_indexing.py]: Do these endpoint calls with `.json` on their result not need `format=json` in the query param part?

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5008844&bid=16522378)

File: src/encoded/tests/test_indexing.py (hunk 1 above)
Comment (j1z0) [inline src/encoded/tests/test_indexing.py]: Codacy Issue found: [Unused wrangler_testapp imported from test_permissions](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5008844)

File: src/encoded/tests/test_indexing.py (hunk 2 above)
Comment (willronchetti) [reply src/encoded/tests/test_indexing.py]: `get` will return a `Response` object who has a `json` member that is the request in JSON form. I do not believe there should be any difference if `format=json` is added or not (just tried it on local 😄 ).

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5008844&bid=16522378)

File: src/encoded/tests/test_indexing.py (hunk 1 above)
Comment (j1z0) [inline src/encoded/tests/test_indexing.py:35]: Codacy Issue found: [Unused wrangler_testapp imported from test_permissions](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5008844)

File: src/encoded/tests/test_indexing.py (hunk 2 above)
Comment (netsettler) [reply src/encoded/tests/test_indexing.py:347]: OK

**Pull request merged**